content/docs/rfcs/14/README.md

```diff
@@ -69,6 +69,13 @@ This provides for asymmetric and symmetric encryption.
 Key agreement is out of band.
 It also provides an encrypted signature and padding for some form of unlinkability.
 
+### Version 2
+
+This indicates that payloads MUST be encoded using [35/WAKU-NOISE](/spec/35).
+
+This provides key-exchanges and encryption based on [Noise Protocol](http://www.noiseprotocol.org/noise.html). 
+It also provides `ChaChaPoly` symmetric encryption.
```

Comment:
In version 1 paragraph we say:

> This provides for asymmetric and symmetric encryption. Key agreement is out of band. It also provides an encrypted signature and padding for some form of unlinkability.

Perhaps we can say high some similar high level stuff? So people who just glance at RFC 14 can get rough idea of why they want to use version 2 over version 1.

Reply:
Addressed in https://github.com/vacp2p/rfc/commit/505e4055ca5537650853d4fd33e7f8bea33650c0
Shall we mention forward secrecy and unlinkability as well, or we just keep it very general as it is now?